src/Value/Lines.php

```diff
@@ -76,18 +80,28 @@ public function seek($line): void
     {
         $this->currentLine = $line;
 
+        try {
+            $this->validate();
+        } catch (\OutOfBoundsException $exception) {
+            --$this->currentLine;
+            throw $exception;
+        }
```

Comment:
this changes the behavior, *but* I think it is more correct.

Reply:
With this change we could remove the `assert($this->isValid())` and I think restoring the old current line would be "more correct".

Imagine we have 5 lines, we calling seek with 3 and then with 55. The current line would be 54 ...

Reply:
Ah, you are right :)